docs/product-user-guide/src/main/asciidoc/decision-tables-defining-proc.adoc

```diff
@@ -0,0 +1,47 @@
+
+.Procedure
+. In a new XLS or XLSX spreadsheet, go to the second or third column and label a cell `RuleSet` (row 9 in example). Reserve the column or columns to the left for descriptive metadata (optional).
+. In the next cell to the right, enter a name for the `RuleSet`. This named rule set will contain all `RuleTable` rules defined in the rule package.
+. Under the `RuleSet` cell, define any rule attributes (one per cell) that you want to apply globally to all rule tables in the package. Specify attribute values in the cells to the right. For supported cell labels and values, see xref:decision-tables-rule-set-entries-ref[].
+. Below the the `RuleSet` area and in the same column as the `RuleSet` cell, skip a row and label a new cell `RuleTable` (row 13 in example) and enter a table name in the same cell. The name string is used as the initial part of the name for all rules derived from this rule table, with the row number appended for distinction. You can override this automatic naming by inserting a `NAME` attribute column.
+. Use the next four rows to define the following elements as needed (rows 14-17 in example):
++
+* *Rule attributes:* Conditions, actions, or other attributes. For supported cell labels and values, see xref:decision-tables-rule-table-entries-ref[].
+* *Object types:* The data objects to which the rule attributes apply. If the same object type applies to multiple columns, merge the object cells into one cell across multiple columns, instead of repeating the object type in multiple cells. When an object type is merged, all columns below the merged range will be combined into one set of constraints within a single pattern for matching a single fact at a time. When an object is repeated in separate columns, the separate columns can create different patterns, potentially matching different or identical facts.
+* *Constraints:* Constraints on the object types.
+* *Column label:* (Optional) Any descriptive label for the column, as a visual aid. Leave blank if unused.
++
+NOTE: As an alternative to populating both the object type and constraint cells, you can leave the object type cell empty and enter the full expression in the constraint cell. For example, instead of `Person` as the object type and `age` as the constraint (separate cells), you can leave the object type cell empty and enter `Person( age == $1)` in the constraint cell.
+
++
+. After you have defined all necessary rule attributes (columns), enter values for each column as needed, row by row, to generate rules (rows 18 and 19 in example). Cells with no data are ignored (such as when a condition or action does not apply).
++
+If you need to add more rule tables to this decision table spreadsheet, skip a row after the last rule in the previous table, label another `RuleTable` cell in the same column as the previous `RuleTable` and  `RuleSet` cells, and create the new table following the same steps in this section.
++
+. Save your XLS or XLSX spreadsheet to finish.
```

Comment:
Should "Save" be bolded? Not sure as this is the excel spreadsheet UI.

Reply:
No. Only bolded when it references the UI, which this doesn't. Just saying generally to save.